components/net_traits/request.rs

```diff
@@ -158,6 +159,7 @@ impl Default for RequestInit {
             referrer_url: None,
             referrer_policy: None,
             pipeline_id: None,
+            redirect_mode: RedirectMode::Manual,
```

Comment:
According to https://fetch.spec.whatwg.org/#concept-request-redirect-mode the default should be Follow.